docs/source/containerization/kubernetes.mdx

```diff
@@ -64,10 +64,10 @@ kind: ServiceAccount
 metadata:
   name: release-name-router
   labels:
-    helm.sh/chart: router-1.0.0-rc.7
+    helm.sh/chart: router-1.0.0-rc.8
```

Comment:
Is it time to make this 1.0? We have had at least one request for this to happen and it's been stable for a while now.

Reply:
Happy to entertain this literally tomorrow.  Feel free to open a PR!